src/backend/updates.rs

```diff
@@ -79,18 +79,11 @@ fn post_upgrade() {
 }
 
 #[allow(clippy::all)]
-fn sync_post_upgrade_fixtures() {
-    mutate(|state| {
-        // Reset Keyda's report again (the last time missed `post_report`, see https://taggr.link/#/post/1273612)
-        if let Some(user) = state.users.get_mut(&3570) {
-            user.post_reports.clear();
-            user.report.take();
-        }
-    })
-}
+fn sync_post_upgrade_fixtures() {}
 
 #[allow(clippy::all)]
 async fn async_post_upgrade_fixtures() {
+    // We need to upgrade the buckets in this release.
```

Comment:
On this note, is this why i'm getting tons of these (since last release):

![image](https://github.com/TaggrNetwork/Taggr/assets/19896126/8604b2d2-5ab9-4abd-840d-008665cb9a88)

Reply:
If you didn't upgrade the buckets - yes.